tests/unit/synapseclient/models/async/unit_test_file_async.py

```diff
@@ -636,7 +636,7 @@ async def test_change_file_metadata(self) -> None:
 
         # WHEN I change the metadata on the example file
         with patch(
-            "synapseclient.models.file.changeFileMetaData",
+            "synapseutils.copy_functions.changeFileMetaData",
```

Comment:
These 2 unit tests broke out of nowhere :/

Reply:
Weird .... What was the issue?

Reply:
I forgot I had made this change here
https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1083/files#diff-cfef7540ef1c528413fd3fcee89094a6344f04634c03fb29cf6c9cfca28a9f93